addons/web_editor/static/src/js/wysiwyg/widgets/image_crop.js

```diff
@@ -205,8 +258,11 @@ export class ImageCrop extends Component {
         });
         delete this.media.dataset.resizeWidth;
         this.initialSrc = await applyModifications(this.media, {forceModification: true, mimetype: this.mimetype});
+        if(this.aspectRatio === "0/0"){
+            cropped = false;
+        }
         this.media.classList.toggle('o_we_image_cropped', cropped);
-        this.$media.trigger('image_cropped');
+        // this.$media.trigger('image_cropped'); // Prevents calling SnippetOptionWidget._rerenderXML.
```

Comment:
Remove this line if it's not needed, but be sure to consider any side effects before removing it.